### PR TITLE
Add Dockerfile and docker-compose.yml to project

### LINKED ISSUE #6: Add Dockerfile and docker-compose.yml to project
Acceptance criteria:
- Add Dockerfile.
- Add docker-compose.yml.

### PR DESCRIPTION
**Pull request by yemelin**
- add Dockerfiles, docker-compose to run in normal and debug mode
- add Makefile
- update Readme

resolve #6

### CODE REVIEW

File: migrations/000001_create_tables.up.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE users (
```
Comment (radisvaliullin) [inline migrations/000001_create_tables.up.sql]: migration files must be in the separate pull requests.

Comment (yemelin) [reply migrations/000001_create_tables.up.sql]: these ones are just demo migrations

File: scripts/pgrunandwait.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+APP_DB_USER=xproject
+APP_DB_PWD=xproject
```
Comment (radisvaliullin) [inline scripts/pgrunandwait.sh]: set sensitive information from env
